api/src/Entity/Day.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Entity;
+
+use ApiPlatform\Core\Annotation\ApiResource;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(uniqueConstraints={
+ *     @ORM\UniqueConstraint(name="offset_period_idx", columns={"periodId", "dayOffset"})
+ * })
+ * @ORM\HasLifecycleCallbacks
+ */
+#[ApiResource]
+class Day extends BaseEntity implements BelongsToCampInterface {
+    /**
+     * @ORM\OneToMany(targetEntity="DayResponsible", mappedBy="day", orphanRemoval=true)
+     *
+     * @var DayResponsible[]
+     */
+    public $dayResponsibles;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="Period", inversedBy="days")
+     * @ORM\JoinColumn(nullable=false, onDelete="cascade")
+     */
+    public ?Period $period = null;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    public int $dayOffset = 0;
+
+    public function getPeriod(): ?Period {
+        return $this->period;
+    }
+
+    public function setPeriod(?Period $period): void {
+        $this->period = $period;
+    }
+
+    public function getCamp(): ?Camp {
+        return (null != $this->period) ? $this->period->camp : null;
+    }
+
+    public function getDayNumber(): int {
```

Comment:
In der alten API hiess dieses Feld nur `number`. Entweder Getter umbenennen oder mit `#[ApiProperty]` in der API umbenennen.